src/scribe_data/checkquery.py

```diff
@@ -40,11 +40,19 @@ class QueryFile:
     def load(self, limit: int) -> str:
         """Load the SPARQL query from 'path' into a string.
 
-        Args:
-            limit (int): the maximum number of results a query should return.
+        Parameters
```

Comment:
Here we need `limit` under `Parameters` as the `load` function takes the argument `limit` above.